setup

```diff
@@ -1,16 +1,22 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I don't know how I feel about setting the shell to bash.

I know some people gave zsh/fsh or other shells as their default and I'm not entirely sure if this would still work in these situations.

Reply:
I am not sure either, as I generally default to bash.
I could revert back to #!/bin/sh
Which in Ubuntu calls dash.
I can then rewrite the string compare to be more compliant with POSIX by using the GNU util 'cut' to shorten the hashes to the significant 32 hexes (and not be bash dependant).